### PR TITLE
feat: Backlog/framework loader henrik

### PR DESCRIPTION
**Pull request by henriknorin-ftrack**
Resolves : 

* CLICKUP-
* FT-
* SENTRY-
* ZENDESK-

- [ ] I have added automatic tests where applicable.
- [ ] The PR contains a description of what has been changed.
- [ ] The description contains manual test instructions.
- [ ] The PR contains update to the release notes.
- [ ] The PR contains update to the documentation.

This PR has been tested on :

- [ ] Windows.
- [ ] MacOs.
- [ ] Linux.

### CODE REVIEW

File: projects/framework-nuke/extensions/nuke.yaml [hunk 1]
```diff
@@ -26,4 +26,19 @@ tools:
...
+    dialog_name: framework_standard_loader_dialog
+    icon: open
+    options:
+      tool_configs:
+        - nuke-image-loader
+      docked: false
+      #event_data:
+      #  selection:
+      #    - entity_id: a66902b4-987c-4ca0-ba39-87be4798f227
+      #      entity_type: component
```
Comment (lluisFtrack) [inline projects/framework-nuke/extensions/nuke.yaml:42]:
```suggestion
```

File: projects/framework-nuke/extensions/tool-configs/nuke-image-loader.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    - ".exr"
+    - ".tif"
+    - ".tiff"
+    - ".tga"
+    - ".bmp"
+    - ".hdr"
+    - ".dpx"
+    - ".cin"
+    - ".psd"
+    - ".tx"
```
Comment (lluisFtrack) [inline projects/framework-nuke/extensions/tool-configs/nuke-image-loader.yaml:18]: Replace it by compatible....

File: projects/framework-nuke/source/ftrack_framework_nuke/__init__.py [hunk 3]
```diff
@@ -176,27 +200,22 @@ def bootstrap_integration(framework_extensions_path):
...
+        startup_tools.append(
+            [
+                name,
+                label,
+                run_on == "startup",
+                action,
+                dialog_name,
+                options,
+            ]
+        )
```
Comment (lluisFtrack) [inline projects/framework-nuke/source/ftrack_framework_nuke/__init__.py:214]: Will improve this with the separated actions, and run_tool. But that will happen on the base branch.

**Review by lluisFtrack — CHANGES_REQUESTED** [review]

**Review by lluisFtrack — CHANGES_REQUESTED** [review]

File: projects/framework-nuke/extensions/tool-configs/nuke-image-loader.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    tags:
+      - context
+    plugin: resolve_entity_path
+    ui: show_component
+
+  - type: plugin
+    tags:
+      - loader
+    options:
+      name: "Nuke image loader"
```
Comment (lluisFtrack) [inline projects/framework-nuke/extensions/tool-configs/nuke-image-loader.yaml:33]: Why are we passing name inside options? Is this really needed?
Can't you put logic inside the plugin or the widget to get the name instead?

**Review by lluisFtrack — CHANGES_REQUESTED** [review]

**Review by lluisFtrack — APPROVED** [review]

File: projects/framework-common-extensions/dialogs/standard_loader_dialog.py [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+        context_plugins = get_plugins(
+            self.tool_config, filters={'tags': ['context']}
+        )
+        for context_plugin in context_plugins:
+            if not context_plugin.get('ui'):
+                continue
+            # Inject the entity data into the context plugin
+            if 'options' not in context_plugin:
+                context_plugin['options'] = {}
+            context_plugin['options'].update(self.dialog_options)
```
[the remark this answers is not in the document]
Comment (henriknorin-ftrack) [reply projects/framework-common-extensions/dialogs/standard_loader_dialog.py:217]: Decided to improve the calls within resolve_entity_paths plugin instead

**Review by lluisFtrack — APPROVED** [review]
Will need to be tested on the final QA

**Pull request merged**